modules/component/spec/core/utils/has-zone.spec.ts

```diff
@@ -1,14 +1,20 @@
 import { hasZone } from '../../../src/core/utils';
-import { NgZone } from '@angular/core';
+import { NgZone, ɵNoopNgZone } from '@angular/core';
 
 class NoopNgZone {}
 
 describe('isZoneLess', () => {
   it('should return false if something else than noop zone is passed', () => {
-    expect(!hasZone({} as NgZone)).toBe(false);
+    expect(
+      !hasZone({
+        runOutsideAngular(fn: Function) {
+          fn.apply(null);
+        },
+      } as NgZone)
+    ).toBe(false);
   });
 
   it('should return true if a noop zone is passed', () => {
-    expect(!hasZone(new NoopNgZone() as NgZone)).toBe(true);
+    expect(!hasZone(new ɵNoopNgZone())).toBe(true);
   });
```

Comment:
I think this one can benefit from refactoring. 
"should return true if a noon zone is passed" - it returns `true` because it's negated with `!hasZone`.

I think this was a result of a change from `isZoneLess` to `hasZone`, which was a good change, however tests were not adjusted accordingly.